jellyfier/Cargo.toml

```diff
@@ -1,34 +1,34 @@
 [package]
 name = "jellyfier"
 version = "0.0.0"
-edition = "2021"
-authors = ["Cardinal"]
 documentation = "https://docs.rs/?"
-homepage = "https://alephzero.org"
 license = "Apache-2.0"
 categories = ["cryptography"]
-repository = "https://github.com/Cardinal-Cryptography/aleph-node"
 description = "Jellyfish PLONK verifier"
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+repository.workspace = true
 
 [dependencies]
+# we must use newer version of arkworks for jellyfish
```

Comment:
Should we understand this comment as "todo: upgrade to newer arkworks version"? 
Currently someone could interpret it as: "0.4.0 below is the newer version, and we must use it".

Reply:
Nvm, I noticed 0.3.0 in workspace, so probably the second meaning was intended.